packages/gimp.rb

```diff
@@ -3,49 +3,90 @@
-    depends_on 'libgudev'
-    depends_on 'libheif'
-    depends_on 'libmng'
-    depends_on 'libtiff'
-    depends_on 'libwmf'
-    depends_on 'llvm'
-    depends_on 'mypaint_brushes'
-    depends_on 'openexr'
-    depends_on 'poppler_data'
-    depends_on 'pygtk'
-    depends_on 'shared_mime_info'
-    depends_on 'xdg_base'
-    depends_on 'sommelier'
-  end
+  compatibility 'all'
+  source_url 'https://download.gimp.org/pub/gimp/v2.10/gimp-2.10.24.tar.bz2'
+  source_sha256 'bd1bb762368c0dd3175cf05006812dd676949c3707e21f4e6857435cb435989e'
 
-  binary_url ({
-    aarch64: 'https://dl.bintray.com/chromebrew/chromebrew/gimp-2.10.18-chromeos-armv7l.tar.xz',
-     armv7l: 'https://dl.bintray.com/chromebrew/chromebrew/gimp-2.10.18-chromeos-armv7l.tar.xz',
-     x86_64: 'https://dl.bintray.com/chromebrew/chromebrew/gimp-2.10.18-chromeos-x86_64.tar.xz',
+  binary_url({
+    aarch64: 'https://dl.bintray.com/chromebrew/chromebrew/gimp-2.10.24-chromeos-armv7l.tar.xz',
+     armv7l: 'https://dl.bintray.com/chromebrew/chromebrew/gimp-2.10.24-chromeos-armv7l.tar.xz',
+       i686: 'https://dl.bintray.com/chromebrew/chromebrew/gimp-2.10.24-chromeos-i686.tar.xz',
+     x86_64: 'https://dl.bintray.com/chromebrew/chromebrew/gimp-2.10.24-chromeos-x86_64.tar.xz'
   })
-  binary_sha256 ({
-    aarch64: '079527b6bb35dbbb7a5b70a8aaae03bb9e90341d1f638e82114c254a35074c9e',
-     armv7l: '079527b6bb35dbbb7a5b70a8aaae03bb9e90341d1f638e82114c254a35074c9e',
-     x86_64: '976807fa73b0f6f3c69750aab669e4ab095f237cdc3970b63e1df50b98f2d619',
+  binary_sha256({
+    aarch64: '6867db24428a1509ac14459f29944a2fbb0a81e8a509b39cd2b7c837fd48fd00',
+     armv7l: '6867db24428a1509ac14459f29944a2fbb0a81e8a509b39cd2b7c837fd48fd00',
+       i686: '6e8faf363dca5a4ae3e1f9eedf05c771016c24256ee67df5a39c586fe565ff0d',
+     x86_64: 'a930aebb265c21d07c8ed0d3e754ca60fe98232335089647d49b294832cdd3d4'
   })
 
+  depends_on 'aalib'
+  depends_on 'alsa_lib'
+  depends_on 'atk'
+  depends_on 'babl'
+  depends_on 'cairo'
+  depends_on 'ffmpeg'
+  depends_on 'fontconfig'
+  depends_on 'freetype'
+  depends_on 'gdk_pixbuf'
+  depends_on 'gegl'
+  depends_on 'gexiv2'
+  depends_on 'ghostscript'
+  depends_on 'gjs'
+  depends_on 'glib'
+  depends_on 'glib_networking'
+  depends_on 'gtk2'
+  depends_on 'harfbuzz'
+  depends_on 'ilmbase'
+  depends_on 'jsonc'
+  depends_on 'json_glib'
+  depends_on 'lcms'
+  depends_on 'libavif'
+  depends_on 'libexif'
+  depends_on 'libgudev'
+  depends_on 'libheif'
+  depends_on 'libice'
+  depends_on 'libjpeg'
+  depends_on 'libmng'
+  depends_on 'libmypaint'
+  depends_on 'libpng'
+  depends_on 'librsvg'
+  depends_on 'libsm'
+  depends_on 'libtiff'
+  depends_on 'libunwind'
+  depends_on 'libwebp'
+  depends_on 'libwmf'
+  depends_on 'libx11'
+  depends_on 'libxcursor'
+  depends_on 'libxext'
+  depends_on 'libxfixes'
+  depends_on 'libxmu'
+  depends_on 'libxpm'
+  depends_on 'libxt'
+  depends_on 'mypaint_brushes'
+  depends_on 'openexr'
+  depends_on 'openjpeg'
```

Comment:
Do openjpeg and libjpeg have conflicting libraries?

Reply:
no